src/Acuminator/Acuminator.Utilities/Roslyn/Semantic/PXContext.cs

```diff
@@ -66,8 +69,6 @@ public class PXContext
 		public INamedTypeSymbol PXGraphExtensionType => Compilation.GetTypeByMetadataName(typeof(PXGraphExtension).FullName);
 		public INamedTypeSymbol PXCacheExtensionType => Compilation.GetTypeByMetadataName(typeof(PXCacheExtension).FullName);
 		public INamedTypeSymbol PXMappedCacheExtensionType => Compilation.GetTypeByMetadataName(typeof(PXMappedCacheExtension).FullName);
-		public INamedTypeSymbol PXViewType => Compilation.GetTypeByMetadataName(typeof(PXView).FullName);
```

Comment:
@metallmonster there was already  nested type for BQL symbols. Maybe these symbols should go there?

Reply:
@SENya1990 yes, that is why I deleted this unnecessary member. You commented on the deleted line.